src/util/string.cpp

```diff
@@ -209,18 +209,25 @@ wchar_t *narrow_to_wide_c(const char *str)
 }
 
 std::wstring narrow_to_wide(const std::string &mbs) {
+#ifdef __ANDROID__
+    return utf8_to_wide(mbs);
```

Comment:
Please use tabulators to indent. Same for L229

Reply:
Oh, thx... Forgot to change profile in my IDE